scaladoc/test/dotty/tools/scaladoc/signatures/SignatureTest.scala

```diff
@@ -75,18 +75,25 @@ abstract class SignatureTest(
   private val unexpectedRegex = raw"(.+)//unexpected".r
   private val identifierRegex = raw"^\s*(`.*`|(?:\w+)(?:_[^\[\(\s]+)|\w+|[^\[\(\s]+)".r
 
-  private def findMissingSingatures(expected: Seq[String], actual: Seq[String]): Set[String] =
+  private def findMissingSignatures(expected: Seq[String], actual: Seq[String]): Set[String] =
     expected.toSet &~ actual.toSet
 
   extension (s: String)
     private def startWithAnyOfThese(c: String*) = c.exists(s.startsWith)
     private def compactWhitespaces = whitespaceRegex.replaceAllIn(s, " ")
 
+  private var counter = 0
```

Comment:
Is the counter necessary here? Or is this just leftover from debugging?

Reply:
Oh sorry, it was used for debugging yes.